services/ui-src/src/utils/other/export.tsx

```diff
@@ -261,6 +271,22 @@ export const getEntityDetailsMLR = (entity: EntityShape) => {
   };
 };
 
+const getNestedIlosResponses = (
+  fieldResponseData: AnyObject,
+  entity: EntityShape
+) => {
+  if (fieldResponseData) {
```

Comment:
`fieldResponseData` is always defined. Is this conditional needed?

Reply:
If the user doesn't "Yes" for ILOS, the `fieldResponseData` could come back undefined because that nested field is never generated, so I just shifted it around to be a check prior to entering this function.